docker-storage-setup.sh

```diff
@@ -940,30 +927,32 @@ if [ -e /etc/sysconfig/docker-storage-setup ]; then
   source /etc/sysconfig/docker-storage-setup
 fi
 
-# Read mounts
-ROOT_DEV=$( awk '$2 ~ /^\/$/ && $1 !~ /rootfs/ { print $1 }' /proc/mounts )
-if ! ROOT_VG=$(lvs --noheadings -o vg_name $ROOT_DEV 2>/dev/null);then
-  Info "Volume group backing root filesystem could not be determined"
-  ROOT_VG=
-else
-  ROOT_VG=$(echo $ROOT_VG | sed -e 's/^ *//' -e 's/ *$//')
-fi
+if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+    # Read mounts
```

Comment:
There are bunch of functionalities this script does which can be independent of driver. For example, growing pvs, growing rootfs, adding devices to volume group etc.

I am especially worried about the use case of growing pv, and rootfs with atomic image in cloud. I would say that let us keep all this in a separate PR and discuss there what can we do.

I think proper solution to this issue is to move all this growing root functionality out of docker-sotrage-setup (may be in cloud-init) and then dss is only responsible for setting up storage for docker and things will become easy.

For now, If you can just retain overlay cleanup related changes, that would be fine.